sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoCoder.java

```diff
@@ -122,6 +126,11 @@
     return of(protoMessageClass);
   }
 
+  public static <T extends Message> ProtoCoder<T> of(
```

Comment:
I'm not very familiar with DynamicMessage usage, but is it actually a `T`? Or is this a `Coder<DynamicMessage>`?

Reply:
Good point... it's indeed a DynamicMessage. Will change.